pkgs/by-name/sb/sbd/package.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sbd";
+  version = "1.36";
+
+  src = fetchurl {
+    url = "https://dl.packetstormsecurity.net/UNIX/netcat/sbd-${finalAttrs.version}.tar.gz";
+    hash = "sha256-oVzkaOfgTMEv4mhuCkThIB18Z5hmgdUeJVy6k2KrBnY=";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace "out = sbd" "sbd = sbd" \
+      --replace \$\(out\) \$\(sbd\)
+  '';
+
+  makeFlags = [
+    "PREFIX=$(out)"
```

```suggestion
    "PREFIX=${placeholder "out"}"
```
if you use placeholder here you can remove `postPatch` as `out` will not conflict with what is used in the Makefile